test/unit/input/files.js

```diff
@@ -61,11 +61,12 @@ suite('Files', function() {
     });
     test('should allow json to override jsonp in 3rd param',
       function(done){
-        var url = 'https://maps.googleapis.com/maps/api/geocode/json?address=Atlanta';
+
+        var url = 'http://localhost:9001/unit/input/array.json';
         var datatype = 'json';
         var myCallback = function(resp){
           assert.ok(resp);
-          //assert.typeOf(resp,'Object');
+          assert.typeOf(resp,'Array');
```

Comment:
Since we have control over the source array, now, should we perhaps add an assertion that `resp.length === 3`? I'm ambivalent but raising it because it occurred to me as a possibility